### PR TITLE
Seabeta 583 feedback form links

### PR DESCRIPTION
**Pull request by CodrinCotarlanSoftwire**

### CODE REVIEW

**Review by DanCorderSoftwire — APPROVED** [review]
As it looks like we'll be removing these feedback forms after private beta this is fine. Otherwise it would be worth trying to find a way to commonise the logic.

File: SeaPublicWebsite.Data/Migrations/SeaDbContextModelSnapshot.cs [hunk 1]
```diff
@@ -281,7 +284,8 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                 {
                     b.HasOne("SeaPublicWebsite.BusinessLogic.Models.PropertyData", null)
                         .WithOne("UneditedData")
-                        .HasForeignKey("SeaPublicWebsite.BusinessLogic.Models.PropertyData", "EditedDataId");
+                        .HasForeignKey("SeaPublicWebsite.BusinessLogic.Models.PropertyData", "EditedDataId")
+                        .OnDelete(DeleteBehavior.Cascade);
```
Comment (DanCorderSoftwire) [inline SeaPublicWebsite.Data/Migrations/SeaDbContextModelSnapshot.cs:288]: Why is this change in this commit? It should already be part of the snapshot file?

File: SeaPublicWebsite.Data/Migrations/SeaDbContextModelSnapshot.cs [hunk 2]
```diff
@@ -38,7 +38,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.HasKey("Id");
 
-                    b.ToTable("DataProtectionKeys");
+                    b.ToTable("DataProtectionKeys", (string)null);
```
Comment (DanCorderSoftwire) [inline SeaPublicWebsite.Data/Migrations/SeaDbContextModelSnapshot.cs]: Why is there no change here removing the `ReturningUser` column? It should have been part of the snapshot file before removing the migration

Comment (CodrinCotarlanSoftwire) [reply SeaPublicWebsite.Data/Migrations/SeaDbContextModelSnapshot.cs]: I took the original file from the main branch and then created the necessary migration. Maybe that is the reason?

File: SeaPublicWebsite.Data/Migrations/SeaDbContextModelSnapshot.cs (hunk 1 above)
Comment (CodrinCotarlanSoftwire) [reply SeaPublicWebsite.Data/Migrations/SeaDbContextModelSnapshot.cs:288]: I'm not sure... unless it wasn't before?

**Pull request merged**